crates/forge/tests/e2e/generate_coverage.rs

```diff
@@ -0,0 +1,21 @@
+use super::common::runner::{setup_package, test_runner};
+use forge_runner::coverage_api::{COVERAGE_DIR, OUTPUT_FILE_NAME};
+
+#[test]
+#[ignore]
```

Comment:
Note: This is ignored until the first release of `cairo-coverage`

Reply:
In order to successfully run the added test, it is also required to use `scarb` version `2.8.0` and update the `Scarb.toml` file in the [simple_package](https://github.com/foundry-rs/starknet-foundry/blob/ddoktorski/cairo-coverage-integration/crates/forge/tests/data/simple_package/Scarb.toml) with:
```
[cairo]
unstable-add-statements-code-locations-debug-info = true
unstable-add-statements-functions-debug-info = true
```